pkg/execution/state/redis_state/queue.go

```diff
@@ -1237,7 +1253,7 @@ func (q *queue) EnqueueItem(ctx context.Context, i QueueItem, at time.Time) (Que
 		partitionTime = q.clock.Now()
 	}
 
-	parts, _ := q.ItemPartitions(ctx, i)
+	parts, _ := q.ItemPartitions(ctx, i, false)
```

Comment:
We do not want to enqueue new items to key queue partitions. This is because we
- significantly grow the global and account partition pointers which may slow down peeks
- spend a lot of resources on peeking and leasing almost-empty partitions vs. processing partitions. QPS is limited and every operation we don't spend on processing actual work is wasted
-